ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -96,7 +104,7 @@ port_profiles:
         # Adapter speed - if not specified will be auto.
       - speed: < interface_speed | forced interface_speed | auto interface_speed >
 
-        # Local endpoint port(s) | required
+        # Interface names of the connected endpoint for documentation | optional
```

Comment:
This is not optional. If you don't specify this the playbook will stop proceeding further. Either we need to specify ```endpoint_ports``` or ```server_ports```. I think ```server_ports``` is a old knob which we are supporting for backward compatibility.

Reply:
You can set this comment to ```# Local endpoint port(s) | required if server_ports is not set```